tests/test_classifier/test_rocauc.py

```diff
@@ -109,6 +109,11 @@ def test_binary_rocauc(self):
 
         # Score the visualizer
         s = visualizer.score(X_test, y_test)
+
+        # Test that score method successfully returns a value between 0 and 1
+        assert 0 <= s <= 1
+
+        # Test that the score method returns a value between 0 and 1
         self.assertAlmostEqual(s, 0.93230159261495249)
```

Comment:
This probably can be removed in favor of the assertion you added.

Reply:
Done